baccmod/base_acceptance_map_creator.py

```diff
@@ -437,15 +437,16 @@ def create_acceptance_map(self, observations: Observations) -> BackgroundIRF:
         """
         pass
 
-    def _compute_dynamic_energy_axis(self, base_energy_axis: MapAxis, data_energy_distribution: np.ndarray, nb_spatial_bin: int) -> MapAxis:
+
+    def _compute_dynamic_energy_axis(self, base_energy_axis: MapAxis, data: np.ndarray, nb_spatial_bin: int) -> MapAxis:
```

Comment:
would be nice to have an option to merge zero or not (likely separate for low and high zero), maybe at the class level ?